src/main/resources/anet-schema.yml

```diff
@@ -476,6 +476,15 @@ properties:
                 title: The long date/time format if `engagementsIncludeTimeAndDuration` is `true`
                 description: Used to show the report engagement date/time in the user interface; should be easy to read, e.g. "Sunday, 6 December 1998 @ 15:30"; see https://momentjs.com/docs/#/displaying/format/ for format specifiers.
 
+  calendarOptions:
+    type: object
+    additionalProperties: false
+    required: [attendeesType]
+    properties:
+      attendeesType:
+        type: string
+        description: The value here determines if advisor names or interlocutors are showed in the rollup calendar
```

Comment:
Suggestion to change the description to: "This value determines which names are shown by default in the report calendar, either advisor or interlocutor."